force-app/main/default/classes/RD2_ChangeView.cls

```diff
@@ -0,0 +1,7 @@
+public with sharing class RD2_ChangeView {
```

Comment:
Could we rename this to `RD2_ChangeHistoryView`? I was trying to think if there would be a more broad use for this class outside of the change history but since we have `statusReason` here that narrows the scope to change history so would be good to have that in the class name for clarity I think.